v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,472 @@
+
+This policy will be used to access CockroachDB client credentials. In keeping with the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), let's give it only the required ability to read the required credential. 
+
+Vault policies are specified using HashiCorp Configuration Language (HCL). The following configuration specifies a policy of read access for the `crdb-role` credential.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault policy write roach-client - <<hcl
+path "database/creds/crdb-role" {
+  capabilities = [ "read" ]
+}
+hcl
+~~~
+
+~~~shell
+Success! Uploaded policy: roach-client
+~~~
+
+### Generate an authentication token for the `crdb-role` Vault user
+
+Our final act as Vault admin will be to provision an authentication token to assume the `crdb-role`, which represents the Vault role with the sole purpose of providing credentials ot a CockroachDB client.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+ vault token create -policy=roach-client
+~~~
+
+~~~txt
+Key                  Value
+---                  -----
+token                hvs.CAESIK4TK7JcSJuKxOgwEa3mYhvfN356Uhikij821K4E4XnWGigKImh2cy5MeUJ5dGNxMjRzNk9qNmVDYWtFYjRUd2QuaU1YdFcQ1cAT
+token_accessor       36udxC5m0hmA0niBYyCThk0k.iMXtW
+token_duration       1h
+token_renewable      true
+token_policies       ["default" "roach-client"]
+identity_policies    []
+policies             ["default" "roach-client"]
+~~~
+
+You can either copy the `token` from the output of the previous command, or capture a token with the following command.
+
+~~~shell
+ VAULT_CLIENT_TOKEN=`vault token create -policy=roach-client -format=json | jq .auth.client_token | tr -d '"'`
+~~~
+
+## Connecting to CockroachDB with Vault-provisioned credentials
+
+### Authenticate to Vault as the CockroachDB user
+
+Use the client token to authenticate to Vault with limited permissions.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault login $VAULT_CLIENT_TOKEN
+~~~
+
+~~~txt
+Success! You are now authenticated. The token information displayed below
+is already stored in the token helper. You do NOT need to run "vault login"
+again. Future Vault requests will automatically use this token.
+~~~
+
+### Confirm the limited permissions of the role
+
+To confirm that you have assumed a role with limited permissions, try listing the current leases on the `crdb-role`, as we did [previously](#). You should see a permissions error.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault list sys/leases/lookup/database/creds/crdb-role/
+~~~
+
+~~~txt
```

Comment:
"You should see a permissions error" You could pull this down to just before the output "You will see a permissions error:" 
Just an opinion, so very much no need to take this, but I try and use "will" instead of conditionals in tutorials. Mainly to instill confidence in the reader and remove any doubt.